yevhenii.dolmatov/task_07/image.rb

```diff
@@ -0,0 +1,30 @@
+require 'faraday'
+require 'open-uri'
+
+class Image
+  def initialize; end
+
+  def download(url)
+    begin
```

Comment:
you should raise an error instead of doing rescue here. if the status is not successful - raise "Wrong URL"
if image extension is not png/jpeg/jpg - raise "wrong type"